internal/jobcontainers/jobcontainer.go

```diff
@@ -237,8 +302,19 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 		envs = append(envs, k+"="+expanded)
 	}
 	env = append(env, envs...)
-
-	env = append(env, sandboxMountPointEnvVar+"="+c.sandboxMount)
+	env = append(env, sandboxMountPointEnvVar+"="+c.rootfsLocation)
+
+	// Add the rootfs location to PATH so you can run things from the root of the image.
+	for idx, envVar := range env {
+		ev := strings.TrimSpace(strings.ToLower(envVar))
+		if len(ev) >= 4 && strings.ToLower(ev[:4]) == "path" {
```

Comment:
If I have an envar like `PathToSomething=foo`, are we going to alter it? Shouldn't we be looking for `path=`?

Reply:
Oh good catch, yes that would happen with the current logic looks like

Reply:
Changed to check for path=